CONTRIBUTING.md

```diff
@@ -38,6 +38,11 @@ To run all tests, use the following command:
 cargo make tests
 ```
 
+Note: On Arch Linux (and presumably a few other Linux Distros), the package for
+Firefox Developer Edition does _not_ create a `firefox` alias, which causes the
+tests to fail unless you also have Firefox installed. To fix this, run
+`sudo ln -s /usr/bin/firefox-developer-edition /usr/bin/firefox`.
```

Comment:
Could you move this to the "Browser tests" section?

I think the phrasing could be a bit more general too (since this isn't really a problem with Arch). Something like this:

> Geckodriver looks for a binary with the name `firefox` in your PATH. If you're using Firefox Developer Edition it might not be able to find it because the binary is named `firefox-developer-edition` on some systems.
In this case you can either download the standard version of Firefox or add a symlink to your PATH.

Feel free to adjust this, I just wanted to give an example.